<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1_btn">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="gBox_btn">
          <property name="title">
           <string>QPushButton</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_1">
             <property name="text">
              <string>按钮1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_2">
             <property name="text">
              <string>按钮2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_3">
             <property name="text">
              <string>按钮3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_4">
             <property name="text">
              <string>按钮4</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QGroupBox" name="gBox_Box">
          <property name="title">
           <string>ComboBox and SpinBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="4">
            <widget class="QFontComboBox" name="fontComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinBox_R"/>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_G"/>
           </item>
           <item row="1" column="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_B"/>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="comboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="gBox_tbtn">
          <property name="title">
           <string>QToolButton</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QToolButton" name="tbtn_1">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="tbtn_2">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="tbtn_3">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="tbtn_4">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="gBox_rbtn">
          <property name="title">
           <string>QRadioButton</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QRadioButton" name="rBtn_1">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="rBtn_2">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rBtn_3">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="rBtn_4">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="gBox_cBox">
          <property name="title">
           <string>QCheckBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QCheckBox" name="cBox_1">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="cBox_2">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="cBox_3">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="cBox_4">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2_text"/>
      <widget class="QWidget" name="page_3_bar"/>
      <widget class="QWidget" name="page_4_etime"/>
      <widget class="QWidget" name="page_5_etc"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>控件</string>
    </property>
    <addaction name="action_btn"/>
    <addaction name="action_text"/>
    <addaction name="action_bar"/>
    <addaction name="action_datetime"/>
    <addaction name="action_etc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_btn">
   <property name="text">
    <string>按钮</string>
   </property>
   <property name="toolTip">
    <string>按钮</string>
   </property>
  </action>
  <action name="action_text">
   <property name="text">
    <string>文本</string>
   </property>
  </action>
  <action name="action_bar">
   <property name="text">
    <string>进度条</string>
   </property>
  </action>
  <action name="action_datetime">
   <property name="text">
    <string>时间日期</string>
   </property>
  </action>
  <action name="action_etc">
   <property name="text">
    <string>其他</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
